### PR TITLE
Customview Ex2

### PR DESCRIPTION
**Pull request by at-lytruong**

### CODE REVIEW

File: ExCustomView/Exercise2/Exercise2/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            view.insertSubview(label, aboveSubview: subView)
+            
+            let button = UIButton()
+            button.frame = CGRect(x: customViewX, y: customViewY, width: widthCustomView, height: 120)
+            button.backgroundColor = UIColor.clear
+
+            button.addTarget(self, action: #selector(clickButton(_:)), for: UIControlEvents.touchUpInside)
+            button.tag = i
+            view.insertSubview(button, aboveSubview: imgView)
+            view.insertSubview(button, aboveSubview: label)
```
Comment (at-truongnguyen) [inline ExCustomView/Exercise2/Exercise2/HomeViewController.swift]: Why we need do it two times?

File: ExCustomView/Exercise2/Exercise2/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+//  ViewController.swift
+//  Exercise2
+//
+//  Created by Hải Lý on 6/14/17.
+//  Copyright © 2017 Hải Lý. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (at-hieutran) [inline ExCustomView/Exercise2/Exercise2/ViewController.swift]: remove unnecessary file. please.

**Comment by at-hieutran** [comment]
@at-lytruong Fix conflict, please.

**Comment by at-truongnguyen** [comment]
Please check my comments

**Pull request merged**